local thank_you_messages = {
    "thank you so much!",
    "thanks a bunch!",
    "thank you for everything!",
    "appreciate it!",
    "thanks a million!",
    "thanks a ton!",
    "much obliged!",
    "thank you kindly!",
    "thanks for your help!",
    "thank you very much!",
    "thanks a heap!",
    "you're the best!",
    "thanks for being awesome!",
    "big thanks!",
    "thanks a bunch!",
    "thanks for all you do!",
    "thanks for being you!",
    "thank you for your support!",
    "thanks a bundle!",
    "you're amazing!",
    "thank you from the bottom of my heart!",
    "thanks for making my day!",
    "thank you for your kindness!",
    "thanks for your generosity!",
    "thank you for your time!",
    "thank you for the lovely item!",
    "wow, this is fantastic! Thank you!",
}

return thank_you_messages